src/elements/Typography.tsx

```diff
@@ -40,6 +40,7 @@ const verticalAlign = style({
 
 export interface TextProps
   extends ColorProps,
+    HTMLAttributes<{}>,
```

Comment:
I thought that @zephraph had already added `className` previously, but I guess that while he did mention the issue it fell through the cracks.

So while this will indeed make general html attributes available from this interface, we may want to decide in the future not to want all of the attributes as `Text` is a primitive and many of these attributes won’t exist on RN.

Having said that, for now you should probably do `HTMLAttributes<HTMLDivElement>` or whatever the underlying tag really is.

Reply:
So, I have a PR open on reaction to help resolve this issue. We actually _don't_ want our Typography components to extend `HTMLAttributes`. This isn't valid in the RN world.